### PR TITLE
Add FAQ response to VRML comparison

### PR DESCRIPTION
**Pull request by jcarpenter**
Per comment question in Dec 28 Road to VR article: http://www.roadtovr.com/mozilla-virtual-reality-webvr-a-frame-guide-basic-vr-website/

### CODE REVIEW

File: src/faq/index.md [hunk 1]
```diff
@@ -49,6 +49,16 @@ There are several community channels available for sharing experiences built A-F
 
 We're easiest to reach on the [A-Frame Slack channel](https://aframevr.slack.com/), via Twitter at [@aframevr](https://twitter.com/aframevr), or by filing issues against the A-Frame [GitHub repo](http://github.com/aframevr/aframe). We love questions, feedback, bug reports and pull requests!
 
+## How is A-Frame different from VRML?
+
+A-Frame is a JavaScript framework—not a web standard. It is built on top of the WebGL API, which is widely supported on modern browsers for desktop and mobile devices. That means it works for developers and users _today_. Browser vendors do not need to know A-Frame exists, or implement support for it.
```
Comment (cvan) [inline src/faq/index.md]: remove the comma: "exists, or implement"

File: src/faq/index.md [hunk 2]
```diff
@@ -49,6 +49,16 @@ There are several community channels available for sharing experiences built A-F
 
 We're easiest to reach on the [A-Frame Slack channel](https://aframevr.slack.com/), via Twitter at [@aframevr](https://twitter.com/aframevr), or by filing issues against the A-Frame [GitHub repo](http://github.com/aframevr/aframe). We love questions, feedback, bug reports and pull requests!
 
+## How is A-Frame different from VRML?
+
+A-Frame is a JavaScript framework—not a web standard. It is built on top of the WebGL API, which is widely supported on modern browsers for desktop and mobile devices. That means it works for developers and users _today_. Browser vendors do not need to know A-Frame exists, or implement support for it. 
+
+This is in contrast to the old approach to implementing new functionality in the web platform, wherein proposal, ratification and implementation of new web standards was a multi-year process with no gaurantee of successful outcome.
```
Comment (cvan) [inline src/faq/index.md]: to be consistent with usage elsewhere, add a serial comma: "ratification, and implementation"

Comment (cvan) [inline src/faq/index.md]: typo: "gaurantee" should be "guarantee"

File: src/faq/index.md [hunk 3]
```diff
@@ -49,6 +49,16 @@ There are several community channels available for sharing experiences built A-F
...
+
+A-Frame embodies the [Extensible Web](https://extensiblewebmanifesto.org/) approach to evolving the web platform, which essentially says, "Instead of taking years to draft and ship web standards that people might not even want, ship new ideas early-and-often as JavaScript libraries/frameworks built atop low-level enabling APIs (like WebGL), let them battle it out in the market, and _then_ look at standardization of the winners".
```
Comment (cvan) [inline src/faq/index.md]: `"Instead"` should be `"instead"`

Comment (cvan) [inline src/faq/index.md]: `early-and-often` should be `early and often`

Comment (cvan) [inline src/faq/index.md]: `like WebGL` should be `such as WebGL`

Comment (cvan) [inline src/faq/index.md]: `of the winners".` should be `of the winners."`

File: src/faq/index.md (hunk 2 above)
Comment (jcarpenter) [inline src/faq/index.md]: Dang it. I need to remember to turn spell check on in Sublime. :stuck_out_tongue_closed_eyes:

File: src/faq/index.md [hunk 4]
```diff
@@ -49,6 +49,16 @@ There are several community channels available for sharing experiences built A-F
...
+
+A-Frame is also notable for it's extensibility and developer ergonomics. It is designed to be familiar to modern web developers and to grow via open source hacking and sharing of community-created components, templates, tools, etc.
```
Comment (cvan) [inline src/faq/index.md]: `it's` should be `its`

File: src/faq/index.md (hunk 1 above)
Comment (cvan) [inline src/faq/index.md]: > framework—not a web standard

I'd perhaps use a comma instead

**Comment by cvan** [comment]
merging; will fix typos

**Pull request merged**